schemas/pa-yaml/v3.0/pa.schema.yaml

```diff
@@ -212,6 +212,8 @@ definitions:
         Layout:
           type: string
           minLength: 1
+        Locked:
```

Comment:
use "IsLocked"

Reply:
Spoken with Joe and Marcel and decided to use Locked since it is shorter and we do not have other boolean properties. I have renamed it to IsLocked for now, we can finalize tomorrow.